hadoop-hdds/container-service/src/test/java/org/apache/hadoop/ozone/container/common/impl/TestContainerDataYaml.java

```diff
@@ -48,6 +49,9 @@
   private static final long MAXSIZE = (long) StorageUnit.GB.toBytes(5);
   private static final Instant SCAN_TIME = Instant.now();
 
+  private static final String VOLUMEOWNER = "hdfs";
+  private static final String CONTAINERDBTYPE = "RocksDB";
+
```

Comment:
NIT: I wold have rather preferred the constant names to be more readable like VOLUME_OWNER, CONTAINER_DB_TYPE

Reply:
thank you for spotting this one, I have fixed this one, and updated the PR.